components/brave_wallet/browser/brave_wallet_p3a.cc

```diff
@@ -309,9 +304,6 @@ void BraveWalletP3A::RecordActiveWalletCount(int count,
     case mojom::CoinType::ZEC:
       histogram_name = kZecActiveAccountHistogramName;
       break;
-    default:
-      NOTREACHED_IN_MIGRATION() << coin_type;
-      return;
   }
```

Comment:
why not drop `default:` but keep NOTREACHED in the last line of the function? just like the guide recommends https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/checks.md#examples
and above